framework/PageDashboard/PageDashboardChart.tsx

```diff
@@ -16,24 +18,66 @@ export function PageDashboardChart(props: {
       value: number;
     }[];
   }[];
+  settings?: {
+    // prevent y axis to show float values
+    onlyIntegerTicks?: boolean;
+    // minimal shown value
+    minDomain?: number | { x?: number; y?: number };
+    // label for x ax
+    yLabel?: string;
+    // label for y ax
+    xLabel?: string;
+    // prevents filtering of zero values
+    allowZero?: boolean;
+    // chart will have lines instead of area
+    useLines?: boolean;
+    // data to be shown in the legend
+    legendData?: {
```

Comment:
The legend should be based off the groups that are passed in by the developer.
That makes it so the dev does not have to keep the legend and groups in sync externally to this component.
SEE: https://github.com/ansible/ansible-ui/pull/781/files#diff-4b17c85c4a9d09cfb0e23bbcc5655cae1662a45515c215ca029119e5fd1d3d0cL43

Reply:
Fixed.